src/sparsevector.jl

```diff
@@ -1558,6 +1596,22 @@ for (fun, mode) in [(:+, 1), (:-, 1), (:*, 0), (:min, 2), (:max, 2)]
     end
 end
 
+for fun in (:+, :-)
+    @eval @propagate_inbounds function $(fun)(x::Union{SparseVectorUnion{Tx},SparseVectorPartialView{Tx}}, y::Union{SparseVectorUnion{Ty},SparseVectorPartialView{Ty}}) where {Tx, Ty}
+        @boundscheck size(x) == size(y)
```

Comment:
This doesn't throw on mismatching size, and I think what you should check is `axes`.

Reply:
Still doesn't throw, you need something like `axes(x) == axes(y) || throw(...)`.